calculator.js

```diff
@@ -6,14 +6,69 @@ const exampleAdditionInput = {
 }
 
 const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+  // console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+  const reg = /\(\d+[-+\/*]\d+\)/;
+  let num1 = '';
+  let num2 = '';
+  if ((reg.test(promptInput.num1) === true) && (reg.test(promptInput.num2) === true)) {
```

Comment:
Wow! Good work stretching yourself and working through this logic. Consider wrapping it up in a function to increase readability.